packages/ui-karma-config/lib/index.js

```diff
@@ -179,13 +180,19 @@ module.exports = function makeConfig({ coverageDirectory, coverageThreshold }) {
           ]
         },
         mode: 'development',
-        devtool: NO_DEBUG ? 'none' : 'inline-source-map',
-
-        //TODO: this is probably a hack and webpack 5 will remove these features
-        //investigate how to do this properly
-        node: { fs: 'empty', module: 'empty' },
+        devtool: NO_DEBUG ? false : 'inline-source-map',
+        plugins: [
+          // fix Buffer/process is not defined errors:
+          new webpack.ProvidePlugin({ process: 'process/browser' }),
+          new webpack.ProvidePlugin({ Buffer: ['buffer', 'Buffer'] })
+        ],
```

Comment:
Webpack 4 was shipped with stubs for some node.js only APIs, these need to be added manually in Webpack 5